src/OrchardCore.Modules/OrchardCore.Navigation/Views/NavigationItemLink.cshtml

```diff
@@ -5,4 +5,4 @@
     Model.Metadata.Alternates.Add("NavigationItemText_Id__" + Model.Id);
 }
 
-<a href="@(Model.Href ?? "#")">@await DisplayAsync(Model)</a>
+<a href="@(Model.Href ?? "#")" target="@Model.Target">@await DisplayAsync(Model)</a>
```

Comment:
For consistency, this should follow the same logic you have in the MenuItemLink.cshtml

Reply:
Oh, that's because I missed MenuItemLink.cshtml.

Reply:
then maybe we should check for empty value then 

target="@(!string.IsNullOrEmpty(Model.Target) ? Model.Target : "_self")"